lib/Dashboard/TalkWidget.php

```diff
@@ -286,6 +286,11 @@ protected function prepareRoom(Room $room, string $userId): WidgetItem {
 		}
 
 		$attendee = $participant->getAttendee();
+
+		if ($attendee->isSensitive()) {
+			$subtitle = '';
+		}
```

Comment:
move before previous if so we save to load, generate and parse the message in first place?